package.json

```diff
@@ -42,7 +42,7 @@
     "workbox-cli": "^4.1.1"
   },
   "dependencies": {
-    "@material-ui/core": "^3.9.3",
+    "@material-ui/core": "^4.1.0",
     "@material-ui/icons": "^3.0.2",
```

Comment:
what about this ?

Reply:
Do you want me to update the icons library as well

Reply:
updated @material-ui/icons to v4.1.0